src/client/tile.h

```diff
@@ -243,6 +244,10 @@ struct TileLayer
 	void applyMaterialOptions(video::SMaterial &material) const
 	{
 		switch (material_type) {
+		case TILE_MATERIAL_SOLID:
```

Comment:
How about `TILE_MATERIAL_OPAQUE` because it is closely related to `TILE_MATERIAL_LIQUID_OPAQUE`?